rll/enable-managed-login.sh

```diff
@@ -195,12 +201,15 @@ enable)
     sudo bash -c "umask 0337 && printf '# Members of the rightscale_sudo group may gain root privileges\n%%rightscale_sudo ALL=(ALL) SETENV:NOPASSWD:ALL\n' > /etc/sudoers.d/90-rightscale-sudo-users"
   fi
 
+  # Ask user input what should be the homedir permission. If no input is given, set default value (0066) which is eq to 7755
```

Comment:
```suggestion
  # Ask user input what should be the homedir permission. If no input is given, set default value (0022) which is eq to 7755
```

Reply:
Also, as in the first suggestion above:
https://github.com/rightscale/rightlink_scripts/pull/124/files#r351433568
`7755 or rwxrwxr-xr-x.` -> `0755 or rwxr-xr-x.`